t/Open/This.t

```diff
@@ -224,6 +224,24 @@ eq_or_diff(
     'spaces in file name but not found'
 );
 
+{
+    my $text = 'a/t/test-data/file';
+    eq_or_diff(
+        parse_text($text),
+        {
+            file_name     => 't/test-data/file',
+            original_text => $text,
+        },
+        'a git-diff path and the file exists'
+    );
+}
+
+eq_or_diff(
+    parse_text('a/t/test-data/i-m-not-here'),
+    undef,
+    'could have been a git diff file name, but it doesn\'t exist'
+);
+
```

Comment:
Should we add a variant of this test for `b` as well?

Reply:
Whatever you prefer. Should we?

Reply:
Please! :)

Reply:
I'm not doing this often enough. How can I ammend the test to the PR?

Reply:
Ah! github magic. Forget the question, please.